livestyled/models/product.py

```diff
@@ -86,12 +86,18 @@ def create_new(
 
     def diff(self, other):
         differences = {}
-        fields = (
-            'external_id', 'price', 'translations', 'product', 'tax', 'tax_rate', 'tax_band'
-        )
+        fields = ('external_id', 'price', 'translations', 'tax', 'tax_rate', 'tax_band')
```

Comment:
Why was this change necessary? product is missing.

Reply:
Because even if these objects body is equal - python will not say that objects are equal if one of them were created within our python sdk and the other one was created within our integration.